jetson/teleop/src/__main__.py

```diff
@@ -257,11 +259,11 @@ def gimbal_control_callback(channel, msg):
     yawData = [keyboardData.a - keyboardData.d,
                keyboardData.j - keyboardData.l]
 
-    gimbal_msg = GimbalCmd()
+    gimbal_msg = MastGimbalCmd()

```

Comment:
i think this gets mad if you dont cast these to floats, but that change should be in #ik-integration